app/src/main/res/values/strings.xml

```diff
@@ -79,174 +79,174 @@
     <string name="new_version_s">New version: %s</string>
     <string name="size_s">Size: %s</string>
     <string name="waiting_for_network">Waiting for network…</string>
     <string name="clear_updates_feed">Clear updates feed</string>
-    <string name="updates_feed_cleared">Updates feed cleared</string>
+    <string name="updates_feed_cleared">Cleared</string>
     <string name="rotate_screen">Rotate screen</string>
     <string name="update">Update</string>
     <string name="feed_will_update_soon">Feed update will start soon</string>
-    <string name="track_sources">Check updates for manga</string>
-    <string name="dont_check">Don`t check</string>
+    <string name="track_sources">Look for updates</string>
+    <string name="dont_check">Don\'t check</string>
     <string name="enter_password">Enter password</string>
     <string name="wrong_password">Wrong password</string>
-    <string name="protect_application">Protect application</string>
-    <string name="protect_application_summary">Ask for password on application start</string>
-    <string name="repeat_password">Repeat password</string>
-    <string name="passwords_mismatch">Passwords do not match</string>
+    <string name="protect_application">Protect the app</string>
+    <string name="protect_application_summary">Ask for password when starting Kotatsu</string>
+    <string name="repeat_password">Repeat the password</string>
+    <string name="passwords_mismatch">Mismatching passwords</string>
     <string name="about">About</string>
     <string name="app_version">Version %s</string>
     <string name="check_for_updates">Check for updates</string>
     <string name="checking_for_updates">Checking for updates…</string>
-    <string name="update_check_failed">Update check failed</string>
+    <string name="update_check_failed">Could not look for updates</string>
     <string name="no_update_available">No updates available</string>
-    <string name="right_to_left">Right to left</string>
-    <string name="prefer_rtl_reader">Prefer Right to left reader</string>
-    <string name="prefer_rtl_reader_summary">You can set up the reading mode for each manga separately</string>
+    <string name="right_to_left">Right-to-left (←)</string>
+    <string name="prefer_rtl_reader">Prefer right-to-left (→) reader</string>
+    <string name="prefer_rtl_reader_summary">Reading mode can be set up separately for each series</string>
     <string name="create_category">New category</string>
     <string name="report_github">Create issue on GitHub</string>
     <string name="scale_mode">Scale mode</string>
     <string name="zoom_mode_fit_center">Fit center</string>
     <string name="zoom_mode_fit_height">Fit to height</string>
     <string name="zoom_mode_fit_width">Fit to width</string>
     <string name="zoom_mode_keep_start">Keep at start</string>
-    <string name="black_dark_theme">Black dark theme</string>
+    <string name="black_dark_theme">Black</string>
```

Comment:
Black _what_?

Reply:
Theme. It is always under that header.
Otherwise it is
```
Theme
Dark theme
Light theme
```
when selecting, and
```
Theme
Dark
```
in the menu.

Reply:
Hm, ok